hieradata/common.yaml

```diff
@@ -458,6 +458,11 @@ govuk::apps::rummager::rabbitmq::enable_publishing_listener: true
 govuk::apps::rummager::redis_host: 'api-redis-1.api'
 govuk::apps::rummager::redis_port: '6379'
 
+govuk::apps::search_admin::db_name: 'search_admin_production'
+govuk::apps::search_admin::db_hostname: 'master.mysql'
+govuk::apps::search_admin::db_password: "%{hiera('govuk::apps::search_admin::db::password')}"
```

Comment:
I think this line isn't necessary because the hiera call has the same name as the hiera value you're setting. So I'd expect this to be either shadowed entirely by the encrypted hieradata, or referring to itself?